### ISSUE TITLE
ModelBuilder won't process training set with unknown error

### ISSUE THREAD

**Issue by Unicodist** [dotnet/machinelearning-modelbuilder#2168]
**Model Builder Version**: _16.7.4.2151101_
**Visual Studion Version**: _2022_

**Bug description**
_ModelBuilder Error_

**Steps to Reproduce**
1. _Add new Machine Learning Model_
2. _Choose Classification_
3. _Use training dataset attached with this issue (Label : Label)_
[Test.csv](https://github.com/dotnet/machinelearning-modelbuilder/files/8920016/Test.csv)

4. _Start Training_

**Expected Experience**
_Expected the system to train the model_

**Actual Experience**
_Encountered error_
![image](https://user-images.githubusercontent.com/47056442/174098541-8f1e6c3b-13f8-4f39-94c4-4f552c9d3570.png)
Log:
[ModelBug.txt](https://github.com/dotnet/machinelearning-modelbuilder/files/8920043/ModelBug.txt)

**Additional Context**
_The problem accured in Asus vivobook - 8 GB RAM - 256 SSD - Processor: AMD Ryzen 5 5500U with Radeon Graphics 2.10 GHz - Windows 11_

**Comment by Unicodist** [comment]
The file I provided seems to be too big to be trained within a minute. I changed training time to 120 seconds and now it works.
